SamplesDashboard/SamplesDashboard/ClientApp/src/views/Details.tsx

```diff
@@ -41,23 +42,32 @@ export default class Details extends React.Component<any, any> {
         const repositoryName = params.name;
         const response = await fetch('api/samples/' + repositoryName);
         const data = await response.json();
-        this.allItems = data;
+        if (data.dependencyGraphManifests.nodes[0]) {
+            this.allItems = data.dependencyGraphManifests.nodes[0].dependencies.nodes;
```

Comment:
>allItems [](start = 17, length = 8)

NIT: not the best name.  something that more closely shows these are dependencyGraohManifest items would probably a little easier to follow.